### PR TITLE
rename to fit class rename

### PR DESCRIPTION
**Pull request by nickdhewitt**

### CODE REVIEW

File: src/main/java/com/kryptnostic/indexing/v1/ObjectSearchPair.java [hunk 1]
```diff
@@ -15,7 +15,7 @@ public ObjectSearchPair( @JsonProperty( INDEX_PAIR_FIELD ) byte[] objectSearchPa
     }
 
     @JsonProperty( INDEX_PAIR_FIELD )
```
Comment (UnsungHero97) [inline src/main/java/com/kryptnostic/indexing/v1/ObjectSearchPair.java]: what about the `@JsonProperty` annotation? this should be `SEARCH_PAIR_FIELD`, right?

**Comment by nickdhewitt** [comment]
For BAC-193

File: src/main/java/com/kryptnostic/indexing/v1/ObjectSearchPair.java (hunk 1 above)
Comment (nickdhewitt) [inline src/main/java/com/kryptnostic/indexing/v1/ObjectSearchPair.java]: good catch. thanks!

Comment (nickdhewitt) [inline src/main/java/com/kryptnostic/indexing/v1/ObjectSearchPair.java]: This will definitely break kryptnostic-js.

I'll add a pr for that too

**Comment by makosblade** [comment]
LGTM, let's wait on the k-js change to be up before we merge it

**Comment by UnsungHero97** [comment]
I'm pretty sure KJS is up to date with this. All KJS cares about is the URL, not the method name.

**Comment by makosblade** [comment]
Right but we updated a JSON field from "indexPair" to "searchPair" so if there's anything in k-js reading that field, it'll need to be updated

**Comment by UnsungHero97** [comment]
right. the JS for `getObjectSearchPair()` is fine, but it needs to be updated for `addObjectSearchPair()`. I have a PR for those changes, so I think this is safe to merge.

https://github.com/kryptnostic/kryptnostic-js/pull/117

**Pull request merged**